### ISSUE TITLE
Music separation failed

### ISSUE THREAD

**Issue by G800807** [intel/openvino-plugins-ai-audacity#45]
Hello. I installed everything correctly according to the instructions in the video. I want to separate a piece of music into instrument tracks, but I get an error message. (Music separation failed)
![error](https://github.com/intel/openvino-plugins-ai-audacity/assets/159252745/49839f39-6834-47a3-9889-6bef1d74ae07)

**Comment by RyanMetcalfeInt8** [comment]
Hi @G800807,

Can you go to Help -> Diagnostics -> Show Log, and copy / paste the contents of the log?

In most cases, reports of this kind of error reveal that the ```openvino-models``` directory was not installed correctly. For example, some users have reported that some zip file extractors will create an extra directory -- and so what ends up being created is, for example, ```C:\Program Files\Audacity\openvino-models\openvino-models``` -- which will cause the plugins to not locate the required model files. 

Thanks,
Ryan

**Comment by G800807** [comment]
Thanks. I see files are missing.

![audacity](https://github.com/intel/openvino-plugins-ai-audacity/assets/159252745/d92f4fc1-2512-4e07-a481-660cfbf833d4)

**Comment by G800807** [comment]
I tried several methods to unpack it, but it didn't work. I was hoping that it would work in Total Commander, but unfortunately it didn't.

**Comment by RyanMetcalfeInt8** [comment]
Hmm, from your log above it seemed like it found the file but is having trouble creating the cache directory. It tries to create this at ```%userprofile%\AppData\Roaming\Audacity\openvino-models```. So in your case, ```C:\Users\GergL\AppData\Roaming\Audacity\openvino-models```. But it seems to be hitting some permission issue... which is strange, as this folder is typically where other files (like config files) are written. 

Can you double check if this directory exists? 
```C:\Users\GergL\AppData\Roaming\Audacity\```

**Comment by G800807** [comment]
I don't have such a folder!

**Comment by RyanMetcalfeInt8** [comment]
Oh wow -- interesting. Can you type ```%appdata%\audacity``` into windows explorer as a path.. does that take you to a folder?

**Comment by G800807** [comment]
Yes!

**Comment by RyanMetcalfeInt8** [comment]
And what is the path to the folder that it takes you to?

**Comment by RyanMetcalfeInt8** [comment]
Oh got it -- There probably is someplace that is not handling this special character correctly. Let me look into this -- I can probably reproduce it by creating a Windows account with a special character.

Thanks very much -- this is very useful info!

**Comment by G800807** [comment]
It was a mistake to use a special character, I know. But so far there has been no problem. I have an old laptop and will give it a try. Thanks for the help, I appreciate it!

**Comment by JaseLovesDub** [comment]
Hi - I'm having the exact same issue; no special characters in the pathway, though!

**Comment by RyanMetcalfeInt8** [comment]
Hi @JaseLovesDub,

So what is the content of the log in your case?

**Comment by JaseLovesDub** [comment]
> Hi @JaseLovesDub,
> 
> So what is the content of the log in your case?

18:05:13: Audacity 3.4.2
18:05:13: Error: Failed to load shared library 'avformat-60.dll' (error 126: The specified module could not be found.)
18:05:13: Error: Failed to load shared library 'avformat-59.dll' (error 126: The specified module could not be found.)
18:05:13: Error: Failed to load shared library 'avformat-58.dll' (error 126: The specified module could not be found.)
18:05:13: Error: Failed to load shared library 'avformat-57.dll' (error 126: The specified module could not be found.)
18:05:13: Error: Failed to load shared library 'avformat-55.dll' (error 193: %1 is not a valid Win32 application.)
18:05:20: sqlite3 message: (1) no such table: project in "SELECT 1 FROM project LIMIT 1;"
18:05:21: Error: Failed to load shared library 'avformat-60.dll' (error 126: The specified module could not be found.)
18:05:21: Error: Failed to load shared library 'avformat-59.dll' (error 126: The specified module could not be found.)
18:05:21: Error: Failed to load shared library 'avformat-58.dll' (error 126: The specified module could not be found.)
18:05:21: Error: Failed to load shared library 'avformat-57.dll' (error 126: The specified module could not be found.)
18:05:21: Error: Failed to load shared library 'avformat-55.dll' (error 193: %1 is not a valid Win32 application.)
18:11:16: Operation 'Export' took 0.119000 seconds. Poll was called 3 times and took 0.000002 seconds. Yield was called 0 times and took 0.000000 seconds.

**Comment by RyanMetcalfeInt8** [comment]
Hi @JaseLovesDub,

Thanks, but I don't any errors in this log... can you open Audacity, run the effect that triggers the issue, and then copy / paste the log?

Thanks,
Ryan

**Comment by JaseLovesDub** [comment]
16:49:19: Audacity 3.4.2
16:49:19: Error: Failed to load shared library 'avformat-60.dll' (error 126: The specified module could not be found.)
16:49:19: Error: Failed to load shared library 'avformat-59.dll' (error 126: The specified module could not be found.)
16:49:19: Error: Failed to load shared library 'avformat-58.dll' (error 126: The specified module could not be found.)
16:49:19: Error: Failed to load shared library 'avformat-57.dll' (error 126: The specified module could not be found.)
16:49:19: Error: Failed to load shared library 'avformat-55.dll' (error 193: %1 is not a valid Win32 application.)
16:49:24: sqlite3 message: (1) no such table: project in "SELECT 1 FROM project LIMIT 1;"
16:49:24: Error: Failed to load shared library 'avformat-60.dll' (error 126: The specified module could not be found.)
16:49:24: Error: Failed to load shared library 'avformat-59.dll' (error 126: The specified module could not be found.)
16:49:24: Error: Failed to load shared library 'avformat-58.dll' (error 126: The specified module could not be found.)
16:49:24: Error: Failed to load shared library 'avformat-57.dll' (error 126: The specified module could not be found.)
16:49:24: Error: Failed to load shared library 'avformat-55.dll' (error 193: %1 is not a valid Win32 application.)
16:49:35: File name is C:\Users\Cex\Music\Various Artists\Ashes to Ashes, Vol. 2\06 Rat Race.mp3
16:49:35: Mime type is *
16:49:35: Opening with libmpg123
16:49:35: Open(C:\Users\Cex\Music\Various Artists\Ashes to Ashes, Vol. 2\06 Rat Race.mp3) succeeded
16:49:36: Operation 'Importing MP3 files' took 0.568000 seconds. Poll was called 7198 times and took 0.013442 seconds. Yield was called 2 times and took 0.005585 seconds.
16:49:43: Operation 'Pre-processing' took 0.000000 seconds. Poll was called 0 times and took 0.000000 seconds. Yield was called 0 times and took 0.000000 seconds.
16:49:43: Error: In Music Separation, exception: Exception from src\inference\src\core.cpp:134:
[ GENERAL_ERROR ] Model file C:\Program Files\Audacity\openvino-models\htdemucs_v4.onnx cannot be opened!

16:49:44: Operation 'OpenVINO Music Separation' took 0.028000 seconds. Poll was called 2 times and took 0.000001 seconds. Yield was called 0 times and took 0.000000 seconds.

**Comment by JaseLovesDub** [comment]
> Hi @JaseLovesDub,
> 
> Thanks, but I don't any errors in this log... can you open Audacity, run the effect that triggers the issue, and then copy / paste the log?
> 
> Thanks, Ryan

Hi Ryan. Thanks for your help.

A clean install seems to have resolved the issue; I suspect there were some older Audacity files that were confusing things.
For anybody else having the same issue, I ran Audacity's uninstaller, was prompted that some files had to be deleted manually, went into the program files directory and wiped it's folder, then wet through a clean install of Audacity, FFmpeg_5.0.0_for_Audacity_on_Windows_x86_64, All the OpenVINO plugs, etc, and Hey Presto.

Happy bunny.

**Comment by G800807** [comment]
Thanks, I'll try to reinstall, but it's a clean install for me. This is a new Windows, it didn't have Audacity.

**Comment by JaseLovesDub** [comment]
> Thanks, I'll try to reinstall, but it's a clean install for me. This is a new Windows, it didn't have Audacity.

Sorry to hear that!

**Comment by art-science-art** [comment]
I'm getting the same error when I try to generate music. I have done a clean reinstall of Audacity, unblocked the zip file, unzipped the files and then copied them into the audacity directory but I get the same error.

**OS Name**	Microsoft Windows 11 Pro
**Version**	10.0.22631 Build 22631
**Processor**	Intel(R) Core(TM) i7-9700F CPU @ 3.00GHz, 3000 Mhz, 8 Core(s), 8 Logical Processor(s)

```
Audacity 3.4.2 64-bit

10:59:04: Audacity 3.4.2
10:59:04: Error: Failed to load shared library 'avformat-60.dll' (error 126: The specified module could not be found.)
10:59:04: Error: Failed to load shared library 'avformat-59.dll' (error 126: The specified module could not be found.)
10:59:04: Error: Failed to load shared library 'avformat-58.dll' (error 126: The specified module could not be found.)
10:59:04: Error: Failed to load shared library 'avformat-57.dll' (error 126: The specified module could not be found.)
10:59:04: Error: Failed to load shared library 'avformat-55.dll' (error 126: The specified module could not be found.)
10:59:07: sqlite3 message: (1) no such table: project in "SELECT 1 FROM project LIMIT 1;"
10:59:07: Error: Failed to load shared library 'avformat-60.dll' (error 126: The specified module could not be found.)
10:59:07: Error: Failed to load shared library 'avformat-59.dll' (error 126: The specified module could not be found.)
10:59:07: Error: Failed to load shared library 'avformat-58.dll' (error 126: The specified module could not be found.)
10:59:07: Error: Failed to load shared library 'avformat-57.dll' (error 126: The specified module could not be found.)
10:59:07: Error: Failed to load shared library 'avformat-55.dll' (error 126: The specified module could not be found.)
10:59:20: Error: In Music Generation, exception: Exception from src\inference\src\core.cpp:134:
[ GENERAL_ERROR ] Model file D:\Program Files\Audacity\openvino-models\riffusion-unet-quantized-int8\unet_int8_sq_0.15_sym_tp_input.xml cannot be opened!

10:59:22: Operation 'OpenVINO Music Generation' took 0.172000 seconds. Poll was called 1 times and took 0.000001 seconds. Yield was called 0 times and took 0.000000 seconds.
```
I'm out of ideas

**Comment by art-science-art** [comment]
Just sharing that I've replicated the issue when the **openvino-models** folder isn't installed into the base directory of Audacity. The "Failed to load shared library 'avformat-60.dll' (error 126" etc. dissappear when the files are installed in the right place.
I had initially copied the contents from inside the openvino-models folder into the main folder of audacity instead of copying the root.

<img width="540" alt="image" src="https://github.com/intel/openvino-plugins-ai-audacity/assets/159601799/eebfb81f-f624-412d-8000-4f6b709f0214">

**Comment by hitesh-geek** [comment]
Hi, its mentioned that  "DefaultCPUAllocator: not enough memory", is that mean i can't run this on i5 7th gen.
Can someone help !!
Thank you
This is the LOG

13:23:19: Audacity 3.4.2
Log Cleared.
13:24:26: Operation 'Pre-processing' took 0.000000 seconds. Poll was called 0 times and took 0.000000 seconds. Yield was called 0 times and took 0.000000 seconds.
13:39:59: Error: In Music Separation, exception: [enforce fail at alloc_cpu.cpp:80] data. DefaultCPUAllocator: not enough memory: you tried to allocate 2468139880 bytes.
00007FFEBE9BB51800007FFEBE9BB3E0 c10.dll!c10::MessageLogger::~MessageLogger [<unknown file> @ <unknown line number>]
00007FFEBE9BBE2300007FFEBE9BBDF0 c10.dll!c10::ThrowEnforceNotMet [<unknown file> @ <unknown line number>]
00007FFEBE9AFFE400007FFEBE9AFE40 c10.dll!c10::alloc_cpu [<unknown file> @ <unknown line number>]
00007FFEBE97805C00007FFEBE978020 c10.dll!c10::DefaultCPUAllocator::allocate [<unknown file> @ <unknown line number>]
00007FFEBE987F1300007FFEBE987EB0 c10.dll!c10::StorageImpl::StorageImpl [<unknown file> @ <unknown line number>]
00007FFE9777F9CF00007FFE9777F280 torch_cpu.dll!at::DynamicLibrary::sym [<unknown file> @ <unknown line number>]
00007FFE97781B0800007FFE977819F0 torch_cpu.dll!at::detail::empty_cpu [<unknown file> @ <unknown line number>]
00007FFE986FD24100007FFE986FCFF0 torch_cpu.dll!at::cpu::set__symint [<unknown file> @ <unknown line number>]
00007FFE977D35EC00007FFE977D3270 torch_cpu.dll!at::TensorIteratorBase::allocate_or_resize_outputs [<unknown file> @ <unknown line number>]
00007FFE977D3C6600007FFE977D3BC0 torch_cpu.dll!at::TensorIteratorBase::build [<unknown file> @ <unknown line number>]
00007FFE977D412C00007FFE977D4020 torch_cpu.dll!at::TensorIteratorBase::build_borrowing_binary_float_op [<unknown file> @ <unknown line number>]
00007FFE9871CC0F00007FFE98709AE0 torch_cpu.dll!at::cpu::where_outf [<unknown file> @ <unknown line number>]
00007FFE986E3B4600007FFE98685560 torch_cpu.dll!at::cpu::bucketize_outf [<unknown file> @ <unknown line number>]
00007FFE982156A500007FFE98215620 torch_cpu.dll!at::_ops::div_Tensor::redispatch [<unknown file> @ <unknown line number>]
00007FFE997F99EA00007FFE99696990 torch_cpu.dll!torch::autograd::CopySlices::~CopySlices [<unknown file> @ <unknown line number>]
00007FFE997EC6CC00007FFE99696990 torch_cpu.dll!torch::autograd::CopySlices::~CopySlices [<unknown file> @ <unknown line number>]
00007FFE9819376200007FFE98193660 torch_cpu.dll!at::_ops::div_Tensor::call [<unknown file> @ <unknown line number>]
00007FFE97722D2400007FFE97722D10 torch_cpu.dll!at::Tensor::div [<unknown file> @ <unknown line number>]
00007FFEBFA0271600007FFEBFA01E60 mod-openvino.dll!c10::ivalue::Object::operator= [<unknown file> @ <unknown line number>]
00007FFEBF9D66AF <unknown symbol address> mod-openvino.dll!<unknown symbol> [<unknown file> @ <unknown line number>]
00007FFED560722900007FFED5601AE6 lib-effects.dll!Effect::LoadUserPreset [<unknown file> @ <unknown line number>]
00007FF74D341F6D00007FF74D0DBA45 Audacity.exe!NoteTrack::AddToDuration [<unknown file> @ <unknown line number>]
00007FF74D3437D800007FF74D0DBA45 Audacity.exe!NoteTrack::AddToDuration [<unknown file> @ <unknown line number>]
00007FF74D2C65B300007FF74D0DBA45 Audacity.exe!NoteTrack::AddToDuration [<unknown file> @ <unknown line number>]
00007FF74D2C670F00007FF74D0DBA45 Audacity.exe!NoteTrack::AddToDuration [<unknown file> @ <unknown line number>]
00007FF74D245C0700007FF74D0DBA45 Audacity.exe!NoteTrack::AddToDuration [<unknown file> @ <unknown line number>]
00007FFEC0A719D400007FFEC0A1C932 wxbase313u_vc_x64_custom.dll!wxBackingFile::operator= [<unknown file> @ <unknown line number>]
00007FFEC0B925AF00007FFEC0A1C932 wxbase313u_vc_x64_custom.dll!wxBackingFile::operator= [<unknown file> @ <unknown line number>]
00007FFEC0B9213C00007FFEC0A1C932 wxbase313u_vc_x64_custom.dll!wxBackingFile::operator= [<unknown file> @ <unknown line number>]
00007FFEC0B93A4D00007FFEC0A1C932 wxbase313u_vc_x64_custom.dll!wxBackingFile::operator= [<unknown file> @ <unknown line number>]
00007FFEC0B9248E00007FFEC0A1C932 wxbase313u_vc_x64_custom.dll!wxBackingFile::operator= [<unknown file> @ <unknown line number>]
00007FFEC0B91B7900007FFEC0A1C932 wxbase313u_vc_x64_custom.dll!wxBackingFile::operator= [<unknown file> @ <unknown line number>]
00007FFEC0B9264400007FFEC0A1C932 wxbase313u_vc_x64_custom.dll!wxBackingFile::operator= [<unknown file> @ <unknown line number>]
00007FFEC0B924EC00007FFEC0A1C932 wxbase313u_vc_x64_custom.dll!wxBackingFile::operator= [<unknown file> @ <unknown line number>]
00007FFEBED4CF5500007FFEBEB9DD3B wxmsw313u_core_vc_x64_custom.dll!wxSizerItemList::size [<unknown file> @ <unknown line number>]
00007FFEC0B9250500007FFEC0A1C932 wxbase313u_vc_x64_custom.dll!wxBackingFile::operator= [<unknown file> @ <unknown line number>]
00007FFEC0B935FA00007FFEC0A1C932 wxbase313u_vc_x64_custom.dll!wxBackingFile::operator= [<unknown file> @ <unknown line number>]
00007FFEBECFE82300007FFEBEB9DD3B wxmsw313u_core_vc_x64_custom.dll!wxSizerItemList::size [<unknown file> @ <unknown line number>]
00007FFEBECFFE6D00007FFEBEB9DD3B wxmsw313u_core_vc_x64_custom.dll!wxSizerItemList::size [<unknown file> @ <unknown line number>]
00007FFEBEF677BB00007FFEBEB9DD3B wxmsw313u_core_vc_x64_custom.dll!wxSizerItemList::size [<unknown file> @ <unknown line number>]
00007FFEBEF6800F00007FFEBEB9DD3B wxmsw313u_core_vc_x64_custom.dll!wxSizerItemList::size [<unknown file> @ <unknown line number>]
00007FFEBEF332C600007FFEBEB9DD3B wxmsw313u_core_vc_x64_custom.dll!wxSizerItemList::size [<unknown file> @ <unknown line number>]
00007FFF0379EF7500007FFF0379E960 USER32.dll!CallWindowProcW [<unknown file> @ <unknown line number>]
00007FFF0379E69D00007FFF0379E1F0 USER32.dll!DispatchMessageW [<unknown file> @ <unknown line number>]
00007FFEBEF3F65D00007FFEBEB9DD3B wxmsw313u_core_vc_x64_custom.dll!wxSizerItemList::size [<unknown file> @ <unknown line number>]
00007FFEC0AAF0C500007FFEC0A1C932 wxbase313u_vc_x64_custom.dll!wxBackingFile::operator= [<unknown file> @ <unknown line number>]
00007FFEC0AAEECE00007FFEC0A1C932 wxbase313u_vc_x64_custom.dll!wxBackingFile::operator= [<unknown file> @ <unknown line number>]
00007FFEC0AAF18C00007FFEC0A1C932 wxbase313u_vc_x64_custom.dll!wxBackingFile::operator= [<unknown file> @ <unknown line number>]
00007FFEC0A74CA600007FFEC0A1C932 wxbase313u_vc_x64_custom.dll!wxBackingFile::operator= [<unknown file> @ <unknown line number>]
00007FF74D17FF5A00007FF74D0DBA45 Audacity.exe!NoteTrack::AddToDuration [<unknown file> @ <unknown line number>]
00007FFEC0AFBB8D00007FFEC0A1C932 wxbase313u_vc_x64_custom.dll!wxBackingFile::operator= [<unknown file> @ <unknown line number>]
00007FFEC0BF76E800007FFEC0A1C932 wxbase313u_vc_x64_custom.dll!wxBackingFile::operator= [<unknown file> @ <unknown line number>]
00007FF74D79A8D600007FF74D0DBA45 Audacity.exe!NoteTrack::AddToDuration [<unknown file> @ <unknown line number>]
00007FFF02C4734400007FFF02C47330 KERNEL32.DLL!BaseThreadInitThunk [<unknown file> @ <unknown line number>]
00007FFF042626B100007FFF04262690 ntdll.dll!RtlUserThreadStart [<unknown file> @ <unknown line number>]

13:40:21: Operation 'OpenVINO Music Separation' took 592.075000 seconds. Poll was called 10287 times and took 3.328390 seconds. Yield was called 2731 times and took 2.004819 seconds.

**Comment by RyanMetcalfeInt8** [comment]
Hi @hitesh-geek,

Hmm, what is the duration of the audio segment that you are trying with? 

Thanks,
Ryan

**Comment by hitesh-geek** [comment]
Its around 2hr 30min audio

**Comment by RyanMetcalfeInt8** [comment]
Ah okay, can you try performing the separation in smaller chunks (say, 10 to 15 mins)? 

Right now the effect isn't designed very well for extremely long segments, as it will want to pull the selected duration into memory at the same time.

 We have some work planned that should fix this, but for now, I would recommend that you try the effect on smaller chunks, and then combine the outputs.

**Comment by hitesh-geek** [comment]
Ok thank you for the info :)

**Comment by jvabbdac** [comment]
Same error here:
12:54:59: Audacity 3.4.2
Log Cleared.
12:55:46: Operation 'Pre-processing' took 0,000000 seconds. Poll was called 0 times and took 0,000000 seconds. Yield was called 0 times and took 0,000000 seconds.
12:55:46: Error: In Music Separation, exception: Exception from src\inference\src\core.cpp:237:
Couldn't create directory [C:\Users\JoÃ£o Vicente], err=Permission denied)

12:55:48: Operation 'OpenVINO Music Separation' took 0,031000 seconds. Poll was called 2 times and took 0,000001 seconds. Yield was called 0 times and took 0,000000 seconds.

Even if I tried a folder without special character or if I add folder permissions 

System: Windows 11 Pro x64, 23H2, OS: 22631.3296, Windows Feature Experience Pack 1000.22687.1000.0;
Specs: Processor: 13th Gen Intel(R) Core(TM) i7-1360P 2.20 GHz, RAM 16,0 GB, NVme 512, no dedicated video

**Comment by RyanMetcalfeInt8** [comment]
Hi @jvabbdac, all,

A fix for this is implemented, it will be included into the upcoming 3.5.0 release. 

Thanks,
Ryan

**Comment by noche99** [comment]
Hi, I also have separation error...

12:46:39: Audacity 3.4.2
12:46:39: Error: Failed to load shared library 'avformat-60.dll' (error 126: The specified module could not be found.)
12:46:39: Error: Failed to load shared library 'avformat-59.dll' (error 126: The specified module could not be found.)
12:46:39: Error: Failed to load shared library 'avformat-58.dll' (error 126: The specified module could not be found.)
12:46:39: Error: Failed to load shared library 'avformat-57.dll' (error 126: The specified module could not be found.)
12:46:39: Error: Failed to load shared library 'avformat-55.dll' (error 126: The specified module could not be found.)
12:46:44: sqlite3 message: (1) no such table: project in "SELECT 1 FROM project LIMIT 1;"
12:46:45: Error: Failed to load shared library 'avformat-60.dll' (error 126: The specified module could not be found.)
12:46:45: Error: Failed to load shared library 'avformat-59.dll' (error 126: The specified module could not be found.)
12:46:45: Error: Failed to load shared library 'avformat-58.dll' (error 126: The specified module could not be found.)
12:46:45: Error: Failed to load shared library 'avformat-57.dll' (error 126: The specified module could not be found.)
12:46:45: Error: Failed to load shared library 'avformat-55.dll' (error 126: The specified module could not be found.)
12:46:57: File name is C:\Users\Snoop\Downloads\Fotos Serenata\Princesita Huanca.mp3
12:46:57: Mime type is *
12:46:57: Opening with libmpg123
12:46:57: Open(C:\Users\Snoop\Downloads\Fotos Serenata\Princesita Huanca.mp3) succeeded
12:46:58: Operation 'Importing MP3 files' took 0.967000 seconds. Poll was called 10757 times and took 0.034195 seconds. Yield was called 6 times and took 0.006918 seconds.
12:47:23: Operation 'Pre-processing' took 0.000000 seconds. Poll was called 0 times and took 0.000000 seconds. Yield was called 0 times and took 0.000000 seconds.
12:47:23: Error: In Music Separation, exception: Exception from src\inference\src\core.cpp:134:
[ GENERAL_ERROR ] Model file C:\Program Files\Audacity\openvino-models\htdemucs_v4.onnx cannot be opened!

12:47:31: Operation 'OpenVINO Music Separation' took 0.030000 seconds. Poll was called 2 times and took 0.000002 seconds. Yield was called 0 times and took 0.000000 seconds.

And also when trying to generate...

12:51:43: Audacity 3.4.2
Log Cleared.
12:52:27: Operation 'Pre-processing' took 0.000000 seconds. Poll was called 0 times and took 0.000000 seconds. Yield was called 0 times and took 0.000000 seconds.
12:52:27: Error: In Music Generation, exception: Exception from src\inference\src\core.cpp:134:
[ GENERAL_ERROR ] Model file C:\Program Files\Audacity\openvino-models\riffusion-unet-quantized-int8\unet_int8_sq_0.15_sym_tp_input.xml cannot be opened!

12:52:29: Operation 'OpenVINO Music Generation' took 0.143000 seconds. Poll was called 1 times and took 0.000001 seconds. Yield was called 0 times and took 0.000000 seconds.

**Comment by McHarun127** [comment]
> Merhaba. Her şeyi videodaki talimatlara göre doğru bir şekilde kurdum. Bir müzik parçasını enstrüman parçalarına ayırmak istiyorum ancak bir hata mesajı alıyorum. (Müzik ayrımı başarısız oldu) ![hata](https://private-user-images.githubusercontent.com/159252745/302951154-49839f39-6834-47a3-9889-6bef1d74ae07.JPG?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTI2MTUwODIsIm5iZiI6MTcxMjYxNDc4MiwicGF0aCI6Ii8xNTkyNTI3NDUvMzAyOTUxMTU0LTQ5ODM5ZjM5LTY4MzQtNDdhMy05ODg5LTZiZWYxZDc0YWUwNy5KUEc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNDA4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDQwOFQyMjE5NDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yZjAxYzU2ZjhkNTcwZDBiOGQzZjk3ODY0ZmY5OTFjOTQ0ZmMwN2NkM2UxNDk3OWFhMDU5Y2I2NmE0ZTljZTk0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.7bsJz52uUK5h2_hXVHd3aSfaHzXLEW18C2wA5udPSJw)




Hello, the cause of the problem has been resolved.
After extracting the openvino-models zip file
The openvino-models folder has 2 nested folders.
Remove the other one and copy it into the files, the problem is solved.
Here is the comment that will help you
_________________________________________________________________________________________________
[Ryguy](https://forum.audacityteam.org/u/Ryguy)
OpenVINO team member
[9d](https://forum.audacityteam.org/t/openvino-error-messages-on-all-tools/102501/6)
It seems to me like the ‘openvino-models’ folder isn’t in the right place, based on the log of the plugin and the output you got from cmd.exe. Double check that this folder exists, and is the one that contains the models (C:\Program Files\Audacity\openvino-models).

A common issue I’ve seen here is that users sometimes extract the zip file in a way that creates a ‘double’ openvino-models folder structure (i.e. C:\Program Files\Audacity\openvino-models\openvino-models).

**Comment by RyanMetcalfeInt8** [comment]
Hi all,

We have an updated release that is posted here: https://github.com/intel/openvino-plugins-ai-audacity/releases/tag/v3.5.0-R2

This should resolve a number of issues, including problems when running under usernames with 'special' characters. We also provide an installer now which will place files into the right spot. 

I'm going to close this issue for now, but feel free to keep commenting if you have questions or want to share results / other feedback. For issues with the new release, I just ask to submit a new ticket -- this will help me stay organized :smile:

Cheers!
